app/api/client/create/route.ts

```diff
@@ -0,0 +1,78 @@
+import dbConnect from "@/app/lib/dbConnect";
+import mongoose from "mongoose";
+import Client from "@/app/models/Client";
+import { ObjectId } from "mongoose";
```

Comment:
Make the following changes:

import { ObjectId } from "mongoose";  → import { Document, Types } from "mongoose";